site/ja/tutorials/images/hub_with_keras.ipynb

```diff
@@ -0,0 +1,1042 @@
+        "plt.ylabel(\"Accuracy\")\n",
+        "plt.xlabel(\"Training Steps\")\n",
+        "plt.ylim([0,1])\n",
+        "plt.plot(batch_stats_callback.batch_acc)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "kb__ZN8uFn-D"
+      },
+      "source": [
+        "### 予測結果を確認する\n",
+        "\n",
+        "前からプロットをやり直すには、まずクラス名の順序付きリストを取得します。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "JGbEf5l1I4jz",
+        "colab": {}
+      },
+      "source": [
+        "class_names = sorted(image_data.class_indices.items(), key=lambda pair:pair[1])\n",
+        "class_names = np.array([key.title() for key, value in class_names])\n",
+        "class_names"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "4Olg6MsNGJTL"
+      },
+      "source": [
+        "モデルを介してイメージバッチを実行し、インデックスをクラス名に変換します。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "fCLVCpEjJ_VP",
+        "colab": {}
+      },
+      "source": [
+        "predicted_batch = model.predict(image_batch)\n",
+        "predicted_id = np.argmax(predicted_batch, axis=-1)\n",
+        "predicted_label_batch = class_names[predicted_id]"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "CkGbZxl9GZs-"
+      },
+      "source": [
+        "結果をプロットします。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "rpFQR1MPMtT1",
+        "colab": {}
+      },
+      "source": [
+        "label_id = np.argmax(label_batch, axis=-1)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "wC_AYRJU9NQe",
+        "colab": {}
+      },
+      "source": [
+        "plt.figure(figsize=(10,9))\n",
+        "plt.subplots_adjust(hspace=0.5)\n",
+        "for n in range(30):\n",
+        "  plt.subplot(6,5,n+1)\n",
+        "  plt.imshow(image_batch[n])\n",
+        "  color = \"green\" if predicted_id[n] == label_id[n] else \"red\"\n",
+        "  plt.title(predicted_label_batch[n].title(), color=color)\n",
+        "  plt.axis('off')\n",
+        "_ = plt.suptitle(\"Model predictions (green: correct, red: incorrect)\")"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "uRcJnAABr22x"
+      },
+      "source": [
+        "## モデルをエクスポートする\n",
+        "\n",
+        "モデルのトレーニングが完了したので、保存されたモデルとしてエクスポートします。"
```

Comment:
[Strong Suggestion]
I think we should not translate `saved model` because it's a technical term which stand for TensorFlow's model format.
I think `saved model` or `SavedModel` is better.